frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -46,258 +28,39 @@ const StyledButton = styled(Button)`
-  font-style: normal;
-  font-weight: normal;
-  font-size: 11px;
-  line-height: 14px;
-`
-
-const DateSelectorGroup = styled.div`
-  padding: 23px 23px 10px 23px;
-  font-family: inter;
-  display: flex;
-  place-content: center;
-  align-self: center;
-  background-color: #fafaf4;
-  border-radius: 15px !important;
-
-  .rdrDateDisplayWrapper {
-    border-radius: 9px !important;
-  }
-`
-
-const EventRightDisplay = styled.div`
-  position: absolute;
-  right: 15px;
-  top: 50%;
-  transform: translate(-17%, -50%);
-  display: flex;
-  align-items: center;
-  justify-content: center;
-`
-
-const BookingSubHeaderCCAName = styled.div`
-  font-style: normal;
-  font-weight: normal;
-  font-size: 13px;
-  line-height: 14px;
-
-  color: rgba(0, 0, 0, 0.65);
-`
-
-const BookingSubHeaderEventName = styled.div`
-  font-style: normal;
-  font-weight: normal;
-  font-size: 13px;
-  line-height: 14px;
-
-  color: rgba(0, 0, 0, 0.65);
-`
-
-const InstructionText = styled.div`
-  font-size: 17px;
-  font-weight: 200;
-  text-align: center;
-  margin-bottom: 15px;
-`
-
 export default function ViewFacility() {
   const dispatch = useDispatch()
   const history = useHistory()
   const params = useParams<{ facilityID: string }>()
-  const {
-    ViewStartDate,
-    ViewEndDate,
-    createSuccess,
-    createFailure,
-    isLoading,
-    facilityBookings,
-    selectedFacilityName,
-    selectedFacilityId,
-  } = useSelector((state: RootState) => state.facilityBooking)
+  const { selectedFacilityName, selectedFacilityId } = useSelector((state: RootState) => state.facilityBooking)
 
   useEffect(() => {
     dispatch(SetIsLoading(true))
     dispatch(fetchFacilityNameFromID(parseInt(params.facilityID)))
-    dispatch(getAllBookingsForFacility(ViewStartDate, ViewEndDate, parseInt(params.facilityID)))
     if (selectedFacilityId == 0) {
       dispatch(setSelectedFacility(parseInt(params.facilityID)))
     }
-    return () => {
-      dispatch(setViewDates(new Date(), parseInt(params.facilityID)))
-    }
   }, [])
 
-  const fetchTelegram = async (booking) => {
-    try {
-      fetch(DOMAIN_URL.FACILITY + ENDPOINTS.TELEGRAM_HANDLE + '/' + booking.userID, {
-        method: 'GET',
-        mode: 'cors',
-      })
-        .then((resp) => resp.json())
-        .then((data) => {
-          if (data.telegramHandle === '' || data.telegramHandle === undefined) {
-            console.log(data.err)
-          } else {
-            openTelegram(data.telegramHandle)
-          }
-        })
-    } catch (err) {
-      console.log(err)
-    }
-  }
-
-  const openTelegram = (userID) => {
-    const site = 'https://telegram.me/' + userID
-    window.open(site)
-  }
-
-  const MyBookingIcon = (
-    <img
-      src={bookingsIcon}
+  const MyBookingButton = (
+    <Button
+      state="primary"
+      text="Bookings"
+      type="button"
+      size="small"
       onClick={() => {
         history.push(PATHS.VIEW_MY_BOOKINGS_USERID + '/' + localStorage.getItem('userID'))
```

Comment:
consider using backticks instead!
```suggestion
        history.push(`${PATHS.VIEW_MY_BOOKINGS_USERID}/${localStorage.getItem('userID')}`)
```